source/common/grpc/async_client_impl.h

```diff
@@ -286,11 +272,11 @@ class AsyncRequestImpl : public AsyncRequest,
       : AsyncStreamImpl<RequestType, ResponseType>(parent, service_method, *this, timeout),
         request_(request), callbacks_(callbacks) {
 
-    current_span_ = parent_span.spawnChild(TracingConfig::get(),
+    current_span_ = parent_span.spawnChild(Tracing::EgressConfig::get(),
```

Comment:
@mattklein123 Was thinking about creating a PR to remove the `Tracing::Config` from the tracer API? It isn't actually used by the tracers themselves, it is used by the code creating spans to determine what tags etc should be added to the spans. Thoughts?

Reply:
I could go either way, no real strong opinion. If you think it's better go for it.

Reply:
Currently there is one place where it is used in the zipkin tracer, to decide whether to used shared span context. Will look to see if there is other ways to do this without passing in the config.